### ISSUE TITLE
getting Index Error when Take-off clicked. 

### ISSUE THREAD

**Issue by KeeseAF** [dcs-liberation/dcs_liberation#1110]
Before filing, please search the issue tracker to see if this feature has already been requested.

**Is your feature request related to a problem? Please describe.**
I mentioned in the Discord server that it was mentioned in BR 641 that cause index errors was due to selling assets opfor was planning against, is it possible that index errors could be cause for bluefor assets as well?

**Describe the solution you'd like**
Before advancing to next day, you could plan out your missions the day before? Just update the opfors mission when you click next day. I don't know about the BTS, but from my *extremely limited* knowledge i Think this makes sense?

**Additional context**
Index error from my last mission . Keep up the great work guys i appreciate the fact that you guys created such an amazing tool that really shouldve been implemented in the game. KUTGW

IndexError: list index out of range
2021-05-23 10:31:50,553 :: INFO :: Generating AirWar (regular)
Traceback (most recent call last):
  File "widgets\QTopPanel.py", line 254, in launch_mission
  File "game\game.py", line 258, in initiate_event
  File "game\event\event.py", line 64, in generate
  File "game\operation\operation.py", line 313, in generate
  File "game\operation\operation.py", line 431, in _generate_transports
  File "gen\convoygen.py", line 31, in generate
  File "gen\convoygen.py", line 34, in generate_convoy
  File "gen\convoygen.py", line 61, in _create_mixed_unit_group
IndexError: list index out of range

**Comment by DanAlbert** [comment]
We can't help you without the data the template asks for. Upload that and we'll reopen.

**Comment by DanAlbert** [comment]
(the bug report template, idk why this was filed as a feature request)

**Comment by KeeseAF** [comment]
oof
my bad

**Comment by DanAlbert** [comment]
If you do have the save then please do. We want to fix bugs, I just can't fix this as reported.

**Comment by Kondor77** [comment]
I'm getting the same error as above, with Liberation 3.0 Release version. I've attached my save file as well as the same error as above. Some context - I just captured an enemy base with an intact factory. My original base seems to be trying to create a convoy of troops to send to the front line, however the newly captured base with factory is handling the front lines now. So my guess is that it sees that, but the convoy it creates is '0' units? 

Traceback (most recent call last):
  File "widgets\QTopPanel.py", line 274, in launch_mission
  File "game\game.py", line 281, in initiate_event
  File "game\event\event.py", line 64, in generate
  File "game\operation\operation.py", line 313, in generate
  File "game\operation\operation.py", line 432, in _generate_transports
  File "gen\convoygen.py", line 31, in generate
  File "gen\convoygen.py", line 34, in generate_convoy
  File "gen\convoygen.py", line 63, in _create_mixed_unit_group
  File "dcs\mission.py", line 682, in vehicle_group
IndexError: list index out of range

[RU11.zip](https://github.com/dcs-liberation/dcs_liberation/files/6688834/RU11.zip)
